tests/Unit/Support/Traits/AssertCssTest.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+namespace Pelago\Tests\Unit\Support\Traits;
+
+use Pelago\Tests\Unit\Support\Traits\Fixtures\TestingAssertCss;
+
+/**
+ * Test case.
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ */
+class AssertCssTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     */
+    public function getCssNeedleRegExpEscapesAllSpecialCharacters()
+    {
+        $needle = '.\\+*?[^]$(){}=!<>|:-/';
+
+        $result = TestingAssertCss::getCssNeedleRegExp($needle);
```

Comment:
You don't need to have a separate class for this - you can directly `use` the trait in this test case class.

Reply:
That did occur to me, though I left it as it was as there could be advantages to having the test and the tested less tightly coupled.  I'll change it.